main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
One of the motivations behind this change is to make the output of `inspect` command roundtrip-able. Displaying `ProjectRef(uri("file:/xxx/helloworld/"),"root")`, the user can put that into either the shell or build.sbt.

It is a bit verbose, but it removes the need to guess or memorize our symbolic conventions. From the output it's clear that the project axis uses `ProjectRef` class.